libraries/app/api.cpp

```diff
@@ -328,6 +328,13 @@ namespace graphene { namespace app {
              start = node.operation_id;
        } catch(...) { return result; }
 
+       if(_app.is_plugin_enabled("elasticsearch")) {
+          auto es = _app.get_plugin<elasticsearch::elasticsearch_plugin>("elasticsearch");
+          auto _thread = std::make_shared<fc::thread>("elasticsearch");
+          return _thread->async([&](){ return es->get_account_history(account, stop, limit, start); },
```

Comment:
Please capture only what's necessary.

Reply:
Also, don't use this if mode is `only_save`.

Reply:
Does this actually work as intended? That would be cool.
We should load-test this before putting it into production.